util/util_linux.go

```diff
@@ -25,7 +26,14 @@ func mountProc() error {
 	return nil
 }
 
-func Mount(device, directory, fsType, options string) error {
+func Mount(device, directory, fsType string, options_i interface{}) error {
```

Comment:
You should be able to do the same with `...string`. `interface{}` is a little overkill here since it'll accept any type.